File: python/ex1.py
class Veiculo:
    def __init__(self, fabricante, tipo, ano_fabricacao):
        self.fabricante = fabricante
        self.tipo = tipo
        self.ano_fabricacao = ano_fabricacao

    def mostrar_informacoes(self):
        return f"Fabricante: {self.fabricante}, Tipo: {self.tipo}, Ano: {self.ano_fabricacao}"


def executar():
    veiculo1 = Veiculo("Toyota", "Corolla", 2024)
    veiculo2 = Veiculo("BMW", "X6", 2023)
    veiculo3 = Veiculo("Chevrolet", "Montana", 2015)

    print(veiculo1.mostrar_informacoes())
    print(veiculo2.mostrar_informacoes())
    print(veiculo3.mostrar_informacoes())


if __name__ == "__main__":
    executar()

File: java/ex1.java
class Veiculo {
    private String marca;
    private String modelo;
    private int ano;

    public Veiculo(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    public void mostrarDetalhes() {
        System.out.println("Marca: " + this.marca + ", Modelo: " + this.modelo + ", Ano: " + this.ano);
    }

    public static void main(String[] args) {
        
        Veiculo veiculo1 = new Veiculo("Toyota", "Corolla", 2020);
        Veiculo veiculo2 = new Veiculo("Honda", "Civic", 2021);
        Veiculo veiculo3 = new Veiculo("Ford", "Focus", 2019);

        
        veiculo1.mostrarDetalhes();
        veiculo2.mostrarDetalhes();
        veiculo3.mostrarDetalhes();
    }
}
